### ISSUE TITLE
A list containing datatypes other than str doesn't work

### ISSUE THREAD

**Issue by abshk-jr** [AtsushiSakai/PyJSONViewer#28]
For the given json file
```json
{
    "character_set_1" : ["1", 2, 3]
}
```
the non-string elements result in an error

![image](https://user-images.githubusercontent.com/55938505/115104806-90d98b00-9f78-11eb-9a1e-0f08d11517a9.png)

**Comment by AtsushiSakai** [comment]
I’m not sure, is it valid JSON? I think all list elements should be same type.

**Comment by abshk-jr** [comment]
It is indeed valid JSON, js treats it as valid too.
Even if one were to use a list with all the elements of the same type, it would still return an error.
The following JSON file doesn't work either
```json
{
    "list" : [1, 2, 3]
}
```

**Comment by AtsushiSakai** [comment]
Thanks. You are right. I hope it is fixed in #30
